src/csharp/Grpc.Core/build/net45/Grpc.Core.targets

```diff
@@ -1,6 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup>
+  <!-- 
+    'SkipGrpcNativeLibsCopying' should not be enabled in normal use.
+
+    It only exists to support special scenarios where user wants to copy the native libraries
+    to output directory themselves, in a separate build step or script.
+
+    If you use this flag, it's your responsibility to ensure that correct versions of 
+    native libraries are used at all times. Mismatching native and managed library
+    versions are not supported.
+  -->
+  <ItemGroup Condition="'$(SkipGrpcNativeLibsCopying)' == ''">
```

Comment:
is `== ''` the right check? if the variable is set to false, this will still be evaluated as true.
Let's follow the msbuild best practices.

Reply:
I changed it to compare against 'true', which works correctly with True, TRUE, true, False, FALSE, false, and whole property missing